docs/spark/how-to-guides/windows-instructions.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Building .NET for Apache Sapark application on Windows
```

Comment:
I'm not going to review this file. The SAME suggestions of ubuntu-instructions can be applied here.
Besides that, I think it's better to combine both articles into one. That way, it'll be easier to maintain, I think.


```suggestion
title: Build .NET for Apache Spark application on Windows
```